app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -52,6 +53,12 @@ def handle_second_factor_locked_user
     render 'two_factor_authentication/shared/max_login_attempts_reached'
   end
 
+  def prevent_current_password_bypass
```

Comment:
would it be possible to take this from a double negative "prevent bypass" to a positive? like `require_current_password` or something?